src/main/java/hudson/plugins/deploy/CargoContainerAdapter.java

```diff
@@ -93,11 +95,17 @@ protected WAR createWAR(File deployableFile) {
         return new WAR(deployableFile.getAbsolutePath());
     }
 
+    /**
+     * Deprecated as of 1.13, please use expandVariable(EnvVars, String)
```

Comment:
I like that you list when it was deprecate and what to use instead (conscientious) but even better would be to include a javadoc link to the new method:  `{@link #expandVariable(EnvVars, String)}`